docs/src/components/pricing/PricingTable.tsx

```diff
@@ -339,19 +339,25 @@ const rowHeaders: Record<string, React.ReactNode> = {
   'MUI Base': (
     <ColumnHead
       label="MUI Base"
-      tooltip="Unstyled components and react hooks available in @mui/base."
+      tooltip="A library of 'unstyled' components and low-level hooks, available @mui/base."
     />
   ),
   'MUI System': (
     <ColumnHead
       label="MUI System"
-      tooltip="CSS utilities for rapidly laying out custom designs available in @mui/system."
+      tooltip="A collection of CSS utilities to help you rapidly lay out custom designs, available @mui/system."
```

Comment:
Why not copy & paste https://mui.com/system/basics/'s description?

```suggestion
      tooltip="CSS utilities for rapidly laying out custom designs available in @mui/system."
```